src/lib.rs

```diff
@@ -114,22 +114,28 @@ fn type_zones(
         .par_iter()
         .map(|z| {
             get_country_code(&country_finder, &z, &country_code, &inclusions[z.id.index])
-                .map(|c| zone_typer.get_zone_type(&z, &c, &inclusions[z.id.index], zones))
+                .map(|c| {
+                    zone_typer.get_zone_type(&z, &c, &inclusions[z.id.index], zones)
+                        .map(|zone_type| (c, zone_type))
+                })
         })
         .collect();
 
     zones
         .iter_mut()
         .zip(zones_type.into_iter())
-        .for_each(|(z, zone_type)| match zone_type {
+        .for_each(|(z, country_code_and_zone_type)| match country_code_and_zone_type {
             None => {
                 info!(
                     "impossible to find a country for {} ({}), skipping",
                     z.osm_id, z.name
                 );
                 stats.zone_without_country += 1;
             }
-            Some(Ok(t)) => z.zone_type = Some(t),
+            Some(Ok((country_code, t))) => {
+                z.country_code = Some(country_code.to_lowercase());
```

Comment:
Very minor: ISO country codes are usually expressed in uppercase. And it also helps to distinguish them from language codes.  
Any particular reason to use lowercase here ?

Reply:
Nope no reason, I'll change it to uppercase 👍🏼. I intended to highlight it as an arbitrary decision on my part open for suggestions. I wasn't aware of the uppercase convention, TIL!